src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -500,7 +500,7 @@ public void Dispose()
                 foreach (var timer in _wakeupTimers.Values)
                 {
                     var handle = new ManualResetEvent(false);
-                    timer.Dispose(handle);
+                    timer.Value.Dispose(handle);
```

Comment:
do we want to check if it was materialized?

Reply:
done